src/main/starlark/builtins_bzl/common/cc/cc_proto_library.bzl

```diff
@@ -293,4 +293,5 @@ cc_proto_library = rule(
             allow_files = False,
         ),
     },
+    toolchains = cc_helper.use_cpp_toolchain(),
```

Comment:
Could you move this to `_cc_proto_aspect`? That should solve the test failure.

Reply:
Thank you! Of course 🤦‍♀️